src/actions/index.js

```diff
@@ -7,3 +7,13 @@ export const loadData = () => {
       .then(data => dispatch({ type: 'DATA_LOADED', payload: data }))
   }
 }
+
+export const searchData = (searchTerm) => {
```

Comment:
My mentor suggested we should consider breaking these calls up, so maybe we should consider it in the future.